### PR TITLE
OCD-3571 - Remove Spring 'acl_' tables

### PR DESCRIPTION
**Pull request by tmy1313**

### CODE REVIEW

**Review by kekey1 — DISMISSED** [review]

File: changes/ocd-3571.sql [hunk 1]
```diff
@@ -0,0 +1,5 @@
+DROP TABLE IF EXISTS openchpl.acl_entry;
```
Comment (andlar) [inline changes/ocd-3571.sql:1]: Are there triggers that need to be removed?

File: dev/openchpl_data-model.sql [hunk 2]
```diff
@@ -3499,6 +3384,7 @@ CREATE TABLE openchpl.certification_criterion_attribute (
   CONSTRAINT certification_criterion_id_fk FOREIGN KEY (criterion_id)
         REFERENCES openchpl.certification_criterion (certification_criterion_id)
         MATCH SIMPLE ON UPDATE NO ACTION ON DELETE RESTRICT
+);
```
Comment (andlar) [inline dev/openchpl_data-model.sql:3387]: Where is this coming from?

Comment (tmy1313) [reply dev/openchpl_data-model.sql:3387]: For some reason I saw it was missing.  Not sure where/when it happened, but figured I'd just fix it.

**Review by andlar — DISMISSED** [review]

**Review by kekey1 — DISMISSED** [review]

**Review by andlar — APPROVED** [review]

**Review by kekey1 — COMMENTED** [review]
Is that create-user function automatically dropped/reloaded in some of our load scripts or does it need to be dropped and reloaded in the ocd sql file that drops the tables?

**Review by kekey1 — APPROVED** [review]

**Comment by andlar** [comment]
> Is that create-user function automatically dropped/reloaded in some of our load scripts or does it need to be dropped and reloaded in the ocd sql file that drops the tables?

The first line of the create-user.sql function drops it, and at least the load script that loads from maintenance executes that sql script before running the function that inserts the users. I think it's close enough to correct that we can test it out and if it's not working somewhere, we'll need to reconfigure something in our build pipeline, as opposed to the code itself

**Pull request merged**